ntoskrnl/cc/cc.cmake

```diff
@@ -0,0 +1,10 @@
+
+list(APPEND SOURCE
```

Comment:
I think that you should not keep using the generic `SOURCE` here, but rather something like `CC_SOURCE` or the likes.